addons/web_editor/static/src/js/wysiwyg/widgets/media.js

```diff
@@ -125,14 +125,12 @@ var FileWidget = SearchWidget.extend({
         this.imagesRows = this.IMAGES_ROWS;
         this.IMAGES_DISPLAYED_TOTAL = this.IMAGES_PER_ROW * this.imagesRows;
 
-        this.options = options;
-        this.context = options.context;
-        this.accept = options.accept;
-
-        this.multiImages = options.multiImages;
+        options = options || {};
+        this.options = _.extend({}, options, {
+            firstFilters: options.firstFilters || [],
+            lastFilters: options.lastFilters || [],
+        });
```

Comment:
I prefer the opposite, not saving the options. But in any case, this should be written like this at least:
```javascript
this.options = _.extend({
    firstFilters: [],
    lastFilters: [],
}, options || {});
```
Only one extend, easier to read and, yes if someone put `firstFilters: true` it will crash at some point... as it should ;)

Reply:
I find it more clear to keep `options` to differentiate which attributes are options and which are not, but maybe this depends on the name of the attributes... In this case I find `this.context` and `this.accept` less than ideal for example.

I agree to only one extend though. Btw the issue is not with `true` (since it would be `true` in the other case too) but with `false`/`undefined` which won't be replaced now. But it doesn't matter because we never pass anything wrong to it.